README.md

```diff
@@ -112,6 +112,27 @@ $token = $session->generateToken(array(
 ));
 ```
 
+## Working with Streams
```

Comment:
Good readme info. Please add phpdoc comments in the source code.

Reply:
@jeffswartz added!